bullet_train/lib/bullet_train/configuration.rb

```diff
@@ -16,6 +16,10 @@ class << self
       def strong_passwords
         @@config&.strong_passwords
       end
+
+      def incoming_webhooks_parent_class_name
+        @@config&.incoming_webhooks_parent_class_name
```

Comment:
```suggestion
        @@config&.incoming_webhooks_parent_class_name || "ApplicationRecord"
```

It might be nice to have a safe default here just in case someone doesn't have the magic line in their initializer. Or maybe we want to set a default in `initialize` instead? (Not sure why we set `@strong_passwords` there. We don't seem to use it.) What do you think, @gazayas?

Reply:
@jagthedrummer Good call, I went ahead and set it as a default value in `initialize`. The original convention of setting `strong_passwords` here was introduced in #23 and I just went along with it (although it wasn't working at the time so I submitted #208).